src/devices/Common/Iot/Device/Common/WeatherHelper.cs

```diff
@@ -13,7 +13,7 @@ namespace Iot.Device.Common
     public static class WeatherHelper
     {
         #region TemperatureAndRelativeHumidity
-        // Formulas taken from https://www.aprweather.com/pages/calc.htm, https://www.wpc.ncep.noaa.gov/html/heatindex_equation.shtml and https://ncalculators.com/meteorology/summer-simmer-index-calculator.htm
+        // Formulas taken from https://www.aprweather.com/pages/calc.htm and https://www.wpc.ncep.noaa.gov/html/heatindex_equation.shtml
```

Comment:
I'm waiting for the response from https://www.aprweather.com/pages/calc.htm owners with clarification on the license

Reply:
Are we going to halt a pull request that took a lot of effort over a line of inline documentation? Can I take it off completely and then decide whether or not to add it in later? @krwq @joperezr

Reply:
I am sorry for the experience and really understand what you are going through, but unfortunately this is not just about the comment, it is about the code using the formulas taken from those sources. For cases like this we really have to wait for a green light from experts in order to be able to go ahead and merge your change. We really appreciate all of the work that has gone into this, and @krwq and I will work hard on trying to figure this out as soon as we can.

Reply:
*bump* :)

Reply:
@joperezr and @krwq are correct. Open source isn't a free for all. You can't just copy & paste code from random projects. The licenses have be to compatible. I'll check what our legal obligations are with respect to math formulas.

Reply:
*bump* @krwq @joperezr @terrajobst